report_thesis/src/sections/background.tex

```diff
@@ -206,7 +206,72 @@ \subsubsection{Support Vector Regression (SVR)}
+
+Decision trees partition the data into subsets based on feature values, aiming to create groups where the data points are more similar in their predicted outcomes. 
+This similarity, or homogeneity, is measured in terms of impurity, which is a measure of how mixed the data points are within a region. 
+This is achieved through recursive binary splitting, dividing the dataset into two regions based on the value of a predictor. 
+The goal at each step is to select the predictor and the split point that result in the greatest reduction in impurity.
+
+Impurity, in the context of decision trees, is measured using the residual sum of squares (RSS). For any predictor $X_j$ and split point $s$, the data is divided into two regions $R1$ and $R2$ defined as:
+$$
+\begin{aligned}
+    R1(j, s) &= \{X | X_j < s\} \\
+    R2(j, s) &= \{X | X_j \geq s\}
+\end{aligned}
+$$
+The goal is to find the values of $j$ and $s$ that minimize the RSS, calculated as:
+$$
+    \sum_{i: x_i \in R1(j, s)} (y_i - \hat{y}_{R1})^2 + \sum_{i: x_i \in R2(j, s)} (y_i - \hat{y}_{R2})^2
+$$
+where $\hat{y}_{R1}$ and $\hat{y}_{R2}$ are the mean responses for the training observations in regions $R1$ and $R2$, respectively.
+
+Once optimal splits are identified, the tree is constructed by repeating this process for each subset until a stopping criterion is met. 
+The final model consists of splits that create distinct regions, each with a predicted response value based on the mean of the observations in that region.
+
+The method ensures that each region is more homogeneous with respect to the target variable, effectively reducing impurity and making the data points within each region more similar.
+
+The predictions from the decision trees can then be improved by growing the trees sequentially, with each new tree correcting the errors of the previous ones. 
+This technique is referred to as \textit{Gradient Boosting}. 
+Each tree is small, with few terminal nodes, preventing large adjustments based on a single tree's predictions. 
+It also ensures that each tree makes small and simple error corrections, such that each step refines the model's performance more reliably.
+Finally, small trees have the property of high bias, meaning that they are unable to capture complex relationships in the data and low variance.
+By combining multiple small trees, the model can capture complex relationships while maintaining a low variance.
+
+The model is constructed iteratively by adding trees to correct the residuals of the previous trees. 
+Initially, the prediction is set as $\hat{f}_0(x) = 0$ and residuals as $r_i = y_i$ for all $i$ in the training set. 
+At each iteration $b$, predictions are updated as:
+$$
+    \hat{f}(x) = \hat{f}(x) + \lambda \hat{f}_b(x)
+$$
+where $\hat{f}_b(x)$ is the prediction of the $b$-th tree and $\lambda$ is the learning rate. Residuals, representing the errors to be corrected by the next tree, are updated as:
```

Comment:
Why define residuals here when you've mentioned it earlier? Should define it when you first mention it.